tests/test_package/test_convert.py

```diff
@@ -0,0 +1,16 @@
+from tidy3d.web.cli import tidy3d_cli
+from tidy3d.web.cli.converter import converter_arg
+from pathlib import Path
+
+
+def test_tidy3d_converter():
+    lsf_path = str(Path.cwd()) + "/tests/data/example.lsf"
```

Comment:
the paths in tests assume you are in the root directory, so
```py
lsf_path = "tests/data/example.lsf"
```
should work. Let's do that for consistency.

Reply:
It appears this doesn't pass the tests. I've added os.getcwd()

Reply:
did you try without the leading `/`? I did this and it worked for me
```py
@pytest.mark.parametrize('filename', ('example.lsf', 'monitors.lsf'))
def test_tidy3d_converter(tmp_path, filename):
    """Test tidy3d convert."""

    lsf_path = f"tests/data/{filename}"
    out_path = str(tmp_path / "out.py")
    converter_arg(lsf_path, out_path)
    assert os.path.exists(out_path)
```